_tests_/GET_all_pixels.spec.ts

```diff
@@ -0,0 +1,62 @@
+import { generateRandomString, generateRandomType } from '../_helpers_/common'
+import { createPixel, listPixels, deletePixel } from '../_helpers_/api.helper'
+import { DEFAULT_TIMEOUT } from '_helpers_/const'
+
+describe('GET /api/pixels', function () {
+    var pixel_id: number[] = []
+
+    test('PIX-001 Verify that user can get list of total pixels', async function () {
+        const record_size = 2;
+        for (let i = 0; i < record_size; i++) {
+            var randomType = generateRandomType();
+            var randomName = generateRandomString();
+            var randomTag = generateRandomString();
+            const newPixel = await createPixel(randomType, randomName, randomTag);
+            console.log(newPixel.body);
```

Comment:
Pls revmove console.log

Reply:
i would like to check if it's successful to create new records

Reply:
you use console.log for debugging purpose. You shouldn't push this.

Reply:
i got it